src/TLabel.cpp

```diff
@@ -30,96 +30,87 @@
 #include "post_guard.h"
 
 
-TLabel::TLabel( QWidget * pW )
-: QLabel( pW )
-, mpHost( 0 )
-, mouseInside()
+TLabel::TLabel(QWidget* pW) : QLabel(pW), mpHost(0), mouseInside()
 {
-    setMouseTracking( true );
+    setMouseTracking(true);
 }
 
 QString nothing = "";
```

Comment:
I don't know. This is a clang-format change. The only thing I did was add `{}` around if statements.